vms/platformvm/state/diff.go

```diff
@@ -330,6 +334,10 @@ func (d *diff) AddChain(createChainTx *txs.Tx) {
 	}
 }
 
+func (d *diff) GetChainSubnet(chainID ids.ID) (ids.ID, error) {
```

Comment:
Why are we allowing calling this method on a `Diff`?

Reply:
In the API it's used over the diff too. See `func (s *Service) chainExists` which is called by `func (s *Service) GetBlockchainStatus` API

Reply:
Got it- I think we should call the parentState here too